inst/htmlwidgets/Widget_BarChart.js

```diff
@@ -35,6 +45,11 @@ HTMLWidgets.widget({
                 let siteSelect;
                 if (x.addSiteSelect)
                     siteSelect = addSiteSelect(el, x.dfSummary, instance, x.siteSelectLabelValue);
+
+                // hide dropdown if in a Shiny environment
+                if (x.bHideDropdown) {
```

Comment:
Does this kick in when the widget is initialized or when the widget is rendered?  I'm not clear on how this setting would be modified app-side.

Reply:
It can be accessed with R using `x` like [this](https://github.com/Gilead-BioStats/gsmApp/blob/main/R/mod_metric_details_server.R#L11)